### ISSUE TITLE
devsim: VkPhysicalDevicePortabilitySubsetPropertiesKHR missing in portability files?

### ISSUE THREAD

**Issue by christophe-lunarg** [LunarG/VulkanTools#1538]
`VkPhysicalDevicePortabilitySubsetPropertiesKHR` and `VkPhysicalDevicePortabilitySubsetFeaturesKHR` are not set in the following files:
portability_minimum_vulkan_1_0.json
portability_minimum_vulkan_1_1.json
portability_minimum_vulkan_1_2.json

**Comment by jeremyk-lunarg** [comment]
These files use the https://schema.khronos.org/vulkan/devsim_1_2_0.json schema, which does not implement the portability subset structs.  The portability subset structs are instead implemented in the https://schema.khronos.org/vulkan/devsim_VK_KHR_portability_subset-provisional-1.json.

I figured a user would use both the minimum portability spec config file with a standard portability subset config file.

**Comment by christophe-lunarg** [comment]
Making it in such a way that it's a possibility, why not, but it seems to me it's really arbitrary and limiting. 

This is so limiting in design that this is not supported by Vulkan Configurator but because devsim support both code path (devsim_1_2_0 and devsim_VK_KHR_portability...) I assume it's a trivial change to make it works. Also devsim_1_2_0.json  schema doesn't prevent adding more structure in the file so we don't have to update the schema.

**Comment by jeremyk-lunarg** [comment]
Currently, devsim only pulls values based on the value of the schemaID.  For example, if the schemaID is the portability schema, it only pulls values for the portability structs.

This pattern was implemented before I started working on the code so I continued it, but now that you mention it, I'm not entirely sure why it doesn't just try and pull every value it can from the config file.

Another improvement to add to the list of things to do.
